### PR TITLE
chore: add-favicon-social-meta-to-library

### PR DESCRIPTION
**Pull request by khawkins98**
* uses vf-favicon
* adds meta attributes

### CODE REVIEW

File: tools/vf-component-library/src/site/_includes/layouts/base.njk [hunk 1]
```diff
@@ -6,6 +6,31 @@
...
+    <meta name="title" content="{{ title or (renderData and renderData.title) or siteConfig.siteInformation.title}}">
+    <meta name="author" content="{{ meta.author or "EMBL" }}">
+    <meta name="robots" content="index, follow">
+    <meta name="keywords" content="{{ meta.keywords }}">
+    <meta name="description" content="{{ meta.description or subtitle}}">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{siteConfig.siteInformation.url}}{{ page.url }}">
+    <meta property="og:title" content="{{ title or (renderData and renderData.title) or siteConfig.siteInformation.title}}">
```
Comment (sturobson) [inline tools/vf-component-library/src/site/_includes/layouts/base.njk:20]: Isn't `siteConfig.siteInformation.title` and such specific to `vf-eleventy`?

Comment (khawkins98) [reply tools/vf-component-library/src/site/_includes/layouts/base.njk:20]: Yeah, this is just for the Eleventy based component library.

Comment (sturobson) [reply tools/vf-component-library/src/site/_includes/layouts/base.njk:20]: 🤦‍♂️

**Review by sturobson — APPROVED** [review]

File: tools/vf-component-library/src/site/_includes/layouts/base.njk (hunk 1 above)
Comment (khawkins98) [reply tools/vf-component-library/src/site/_includes/layouts/base.njk:20]: 😄

**Pull request merged**